about.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./css/about.css">
+  <link rel="stylesheet" href="./css/style.css">
+  <script src="https://kit.fontawesome.com/035535dd6b.js" crossorigin="anonymous"></script>
+  <title>GH Comedy Crusade</title>
+</head>
+<body>
+  <!-- Social Navigation -->
+  <nav class="nav">
+    <div class="container">
+      <div class="nav__social">
+        <ul class="nav__link">
+          <li><a href="#" class="nav__link--items"><i class="fa-brands fa-facebook-f"></i></a></li>
+          <li><a href="#" class="nav__link--items"><i class="fa-brands fa-twitter"></i></a></li>
+          <li><a href="#" class="nav__link--items">English</a></li>
+          <li><a href="#" class="nav__link--items">Blog</a></li>
+          <li><a href="#" class="nav__link--items">Logout</a></li>
+        </ul>
+      </div>
+    </div>
+  </nav>
+  <!-- End of social Navigation -->
+  
+  <!-- Menu Navigation -->
+  <div class="background">
+    <nav class="menu">
+      <div class="container menu__container">
+        <div class="menu__logo">
+          <a href="./index.html"><img src="./images/logo.png" alt="GH comedy crusade logo" class="menu__logo--img"></a>
+          <a href="#" class="menu__logo--hamburger"><i class="fa-solid fa-bars fa-2x"></i></a>
+      </div>
+      <ul class="menu__link">
+        <li><a href="#" class="menu__link--items active">About</a></li>
+        <li><a href="#" class="menu__link--items">Program</a></li>
+        <li><a href="#" class="menu__link--items">Join</a></li>
+        <li><a href="#" class="menu__link--items">Sponsor</a></li>
+        <li><a href="#" class="menu__link--items">News</a></li>
+        <li><a href="#" class="menu__link--campaign">Book a Comedian</a></li>
+      </ul>
+    </div>
+  </nav>
```

Comment:
- [ ] You did an amazing job creating your navbar :+1: :1st_place_medal:, but I think it would be better if you add a link from the about page to the main page for the mobile users because it would improve the UX :100: